client/src/main/java/org/schemarepo/client/Avro1124RESTRepositoryClient.java

```diff
@@ -93,7 +93,7 @@ public Subject lookup(String subject) {
   public Iterable<Subject> subjects() {
     ArrayList<Subject> subjectList = new ArrayList<Subject>();
     try {
-      String subjects = webResource.get(String.class);
+      String subjects = webResource.accept(MediaType.TEXT_PLAIN_TYPE).get(String.class);
```

Comment:
Ugh... this is getting complicated. Just how widespread do you think the use of old AVRO-1124 client is? Are we jumping through hoops b/c of a few pieces of anecdotal evidence or there's something more substantial? To be clear, I'm not challenging your statement, just want to have a better underlying picture.

Now to the tech side of it... One thing I can do is to look at User-Agent header. Presumably jersey won't have it set at all (or set to a specific value) whereas a real browser will of course always have it. This smells of a kludge but seems to me a "lesser evil". I'll report back once I reconfirm my assumptions regarding UA.